Components/VoteCount/vote.jsx

```diff
@@ -0,0 +1,50 @@
+import { fetchArticleByID, updateVotes } from "../../Axios Requests/requests";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import '../VoteCount/vote.css'
+
+export default function Count(){
+    const [ likeCount, setLikeCount] = useState(0)
+    const { article_id } = useParams()
+
+    function handleLike(){
+        const newLikeCount = likeCount + 1
+        setLikeCount(newLikeCount)    
+        updateVotes(article_id, 1).then(response => {
+        setLikeCount(response.data.votes)
+        }).catch(error => {
+            console.error('Vote Not Registered')
```

Comment:
Instead of logging this error, please display a message to the user that something went wrong.